pom.xml

```diff
@@ -61,7 +61,7 @@
     <revision>4.11.4</revision>
     <changelist>-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
-    <jenkins.version>2.289.1</jenkins.version>
+    <jenkins.version>2.303.3</jenkins.version>
```

Comment:
Minimum version for deps.

Reply:
https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/119#issuecomment-1136531975 however